doc/tutorial/Authentication.lhs

```diff
@@ -281,17 +281,22 @@ lookupAccount key = case Map.lookup key database of
 For generalized authentication, servant exposes the `AuthHandler` type,
 which is used to wrap the `Request -> Handler user` logic. Let's
 create a value of type `AuthHandler Request Account` using the above `lookupAccount`
-method:
+method (note: we depend upon `cookie`'s `parseCookies` for this):
```

Comment:
I think it's the contrary, you have to do a link around `text`, like this: [`servant`](https://github.com/haskell-servant/servant)